src/vs/workbench/contrib/files/browser/views/explorerViewer.ts

```diff
@@ -67,6 +67,7 @@ import { IHoverService } from 'vs/workbench/services/hover/browser/hover';
 import { HoverPosition } from 'vs/base/browser/ui/hover/hoverWidget';
 import { IFilesConfigurationService } from 'vs/workbench/services/filesConfiguration/common/filesConfigurationService';
 import { mainWindow } from 'vs/base/browser/window';
+import { TestCoverageBars } from 'vs/workbench/contrib/testing/browser/testCoverageBars';
```

Comment:
I think conceptually `contrib/files` should not have a dependency to `contrib/testing`